src/site/antora/modules/ROOT/pages/manual/appenders/network.adoc

```diff
@@ -0,0 +1,980 @@
+----
+
+Properties::
++
+.Snippet from an example {antora-examples-url}/manual/appenders/network/http.properties[`log4j2.properties`]
+[source,properties,indent=0]
+----
+include::example$manual/appenders/network/http.properties[tag=appender]
+----
+====
+
+<1> This HTTP header is evaluated once at configuration time.
+<2> This HTTP header is evaluated at each log event.
+
+[[SMTPAppender]]
+== SMTP Appender
+
+The SMTP writes log events to an e-mail service using
+https://jakarta.ee/specifications/mail/1.6/[Jakarta Mail 1.6 or higher].
+
+[IMPORTANT]
+====
+Unlike what happens with
+xref:manual/filters.adoc#appender-stage[most appenders], the SMTP does **not** discard log events that are denied by the
+<<SmtpAppender-element-Filter,`Filter`>> element.
+
+All the log events received by the appender are added to a cyclic log event buffer as context.
+If the filter accepts a message, an e-mail is sent.
+====
+
+[#SmtpAppender-attributes]
+.SMTP Appender configuration attributes
+[cols="1m,1,2,5"]
+|===
+| Attribute | Type | Default value | Description
+
+4+h| Required
+
+| [[SmtpAppender-attr-name]]name
+| `String`
+|
+| The name of the appender.
+
+4+h| Optional
+
+| [[SmtpAppender-attr-bufferSize]]bufferSize
+| `int`
+| `512`
+|
+The maximum number of log events to be buffered for inclusion in a single message.
+
+| [[SmtpAppender-attr-ignoreExceptions]]ignoreExceptions
+| `boolean`
+| `true`
+| If `false`, logging exception will be forwarded to the caller of the logging statement.
+Otherwise, they will be ignored.
+
+Logging exceptions are always also logged to xref:manual/status-logger.adoc[]
+
+| [[SmtpAppender-attr-smtpDebug]]smtpDebug
+| `boolean`
+| `false`
+|
+When set to `true` turns on the session debugging.
+In the reference implementation of Jakarta Mail
+(https://projects.eclipse.org/projects/ee4j.angus[Eclipse Angus])
+this will cause log messages to be printed on `System.out`.
+
+See the javadoc of
+https://jakarta.ee/specifications/mail/2.1/apidocs/jakarta.mail/jakarta/mail/session#setDebug(boolean)[`Session.setDebug()`]
+for more details.
+
+| [[SmtpAppender-attr-]]smtpProtocol
+| `String`
+| `smtp`
+a|
+The Jakarta Mail transport protocol.
+Most implementations provide:
+
+`smtp`::
+SMTP
+
+`smtps`::
+SMTP over TLS
+
+| [[SmtpAppender-attr-smtpHost]]smtpHost
+| `String`
+| `localhost`
+|
+The SMTP hostname to send to.
+
+| [[SmtpAppender-attr-Port]]smtpPort
+| `int`
+a|
+`25`:: for `smtp`
+`465`:: for `smtps`
+|The SMTP port to send to.
+
+| [[SmtpAppender-attr-smtpUsername]]smtpUsername
+| `String`
+|
+|
+The username used to authenticate against the SMTP server.
+
+| [[SmtpAppender-attr-smtpPassword]]smtpPassword
+| `String`
+|
+|The password used to authenticate against the SMTP server.
+
+
+4+h| https://datatracker.ietf.org/doc/html/rfc5322[RFC 822 message] fields
+
+| [[SmtpAppender-attr-from]]from
+| https://jakarta.ee/specifications/mail/2.1/apidocs/jakarta.mail/jakarta/mail/internet/internetaddress[`InternetAddress[]`]
```

Comment:
Array notations break the AsciiDoc syntax:
![image](https://github.com/user-attachments/assets/8ed80b55-7dc7-441c-bdf3-d928c7938229)

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2796/commits/e53b3dd39b7e3686d59f25062f4642d9f515673f